test/SoftMax-CrossEntropyTest.jl

```diff
@@ -0,0 +1,82 @@
+include("../src/layers/LayerBase.jl")
+include("../src/layers/SoftMaxCrossEntropy.jl")
+include("../src/layers/SoftMax.jl")
+include("../src/layers/CrossEntropyLoss.jl")
+using Base.Test
+
+smaxcross = SoftMaxCrossEntropyLoss()
```

Comment:
Resolved by creating separate tests for `SoftMax` and `CrossEntropyLoss` as mentioned in other reviews; `SoftMax-CrossEntropyTest.jl` is technically not needed and will be deleted in next commit unless otherwise requested.